test/e2e/tests/ppom/ppom-blockaid-alert-erc20-transfer.spec.ts

```diff
@@ -90,13 +90,19 @@ describe('PPOM Blockaid Alert - Malicious ERC20 Transfer', function (this: Suite
           .withPreferencesController({
             securityAlertsEnabled: true,
           })
+          .withEnabledNetworks({
+            eip155: {
+              '0x1': true,
+            },
+          })
+          .withNetworkControllerOnMainnet()
           .build(),
         testSpecificMock: mockInfuraWithMaliciousResponses,
         title: this.test?.fullTitle(),
       },
 
       async ({ driver }) => {
-        await loginWithBalanceValidation(driver);
+        await loginWithoutBalanceValidation(driver);
```

Comment:
sucks that we are removing balance validation assertions. Will need to discuss with QA on how we can add this back in.